src/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,19 @@
+import { NAV_LOGO } from "src/utils/constants/constants";
+import Search from "src/components/search/Search";
+import "src/components/navbar/Navbar.component.scss";
+
+const Navbar = () => {
+  return (
+    <div className="navbar">
+      <div className="navbar-logo">
+        <img src={NAV_LOGO} alt="Logo" />
```

Comment:
Please convert all the text into constant and import them instead
```suggestion
        <img src={NAV_LOGO} alt={ALT_TEXT_LOGO} />
```